assets/app/views/edit/build-config.html

```diff
@@ -0,0 +1,526 @@
+<project-header class="top-header"></project-header>
+  <project-page>
+    <!-- Middle section -->
+    <div class="middle-section" ng-if="buildConfig">
```

Comment:
We should look at the problem with the key/value directive that causes the error, but that is unrelated to your change and can be a follow on.